pkg/session_recording/recordings.jsx

```diff
@@ -543,14 +524,11 @@
         }
 
         render() {
-            const dateSince = this.state.dateSince;
-            const dateUntil = this.state.dateUntil;
-
             if (this.state.recordingID === null) {
                 return (
                     <RecordingList
-                        dateSince={dateSince} onDateSinceChange={this.handleDateSinceChange}
-                        dateUntil={dateUntil} onDateUntilChange={this.handleDateUntilChange}
+                        onDateSinceChange={this.handleDateSinceChange}
+                        onDateUntilChange={this.handleDateUntilChange}
```

Comment:
I understand you wanted to simplify the code and I agree less code is neater, but we still have to keep the date range in the URL, so that people could make bookmarks. The URL needs to be handled by the View, and it will need to pass the initial values from the URL down to the Datepickers. I'm not sure we can avoid having to pass the dates all the way from the View. Or did you have some other idea about this?